processing/src/main/java/io/druid/query/aggregation/JavaScriptAggregatorFactory.java

```diff
@@ -172,7 +172,7 @@ public AggregatorFactory getMergingFactory(AggregatorFactory other) throws Aggre
               @Override
               public AggregatorFactory apply(String input)
               {
-                return new JavaScriptAggregatorFactory(input, fieldNames, fnAggregate, fnReset, fnCombine, config);
+                return new JavaScriptAggregatorFactory(input, Lists.newArrayList(input), fnCombine, fnReset, fnCombine, config);
```

Comment:
Previously getCombiningFactory was always called on top of getRequiredColumn to get the identity AggregatorFactory needed for copying the javascript aggregator values. Since getCombiningFactory is no longer called on this, the original getRequiredColumn does really make sense for copying values.